### PR TITLE
Inf update

### PR DESCRIPTION
**Pull request by fungunga**
Huhu! We re-modelled some dependencies and added (thanks Mathias!) a YAML validation to find dependency errors BEFORE running in the good old div-by-zero error.

### CODE REVIEW

File: _services/cosmotool.yaml [hunk 1]
```diff
@@ -1,8 +1,7 @@
 ---
-title: 'Data Federation Architecture (Cosmotool)'
-description: 'DFA Cosmotool'
-website: 'https://dfa.de.dariah.eu/'
+title: 'DARIAH-DE Cosmotool'
+description: 'DARIAH-DE Cosmotool'
```
Comment (mathias-goebel) [inline _services/cosmotool.yaml]: description MUST BE not equal to title.

File: _servers/annotation.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
```
Comment (mathias-goebel) [inline _servers/annotation.yaml:1]: ok, you are moving this file, but what about the _service_?

File: _services/sp.textgridlab.org.yaml [hunk 3]
```diff
@@ -5,4 +5,5 @@ website: 'https://sp.textgridlab.org/'
 dependencies:
   - '/infrastructure/gwdg-cloud'
   - '/servers/dfa'
+# ??????????????? Wird der hier benötigt im Status-Modell ???????????????????????
```
Comment (mathias-goebel) [inline _services/sp.textgridlab.org.yaml]: i suggest leaving a `TODO` or `FIXME` marker here, but not a bunch of question marks.

Comment (mathias-goebel) [reply _services/sp.textgridlab.org.yaml]: or may be even better: moving this [here](https://github.com/DARIAH-DE/status/issues/new/choose)

Comment (fungunga) [reply _services/sp.textgridlab.org.yaml]: done

File: _services/cosmotool.yaml (hunk 1 above)
Comment (fungunga) [reply _services/cosmotool.yaml]: done

**Review by ubbo — APPROVED** [review]

**Pull request merged**